test/y2network/sysconfig/connection_config_writers/tun_test.rb

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "../../../test_helper"
+
+require "y2network/sysconfig/connection_config_writers/tun"
+require "y2network/startmode"
+require "y2network/boot_protocol"
+require "y2network/connection_config/tun"
+require "y2network/connection_config/ip_config"
+
+describe Y2Network::Sysconfig::ConnectionConfigWriters::Tun do
+  subject(:handler) { described_class.new(file) }
+
+  def file_content(scr_root, file)
```

Comment:
Maybe we should start thinking about a helper (we have the same method in many places).

Reply:
Definitely

Reply:
Will try to do it later when implementing a new connection type, probably with the bonding one.